src/components/atoms/FileInput/FileInput.tsx

```diff
@@ -125,9 +127,15 @@ export default function FileInput<TFile extends UploadFile = UploadFile>({
           type="file"
           {...props}
         />
-        <span className={clsx(styles.placeholder, innerClassNames.input)}>
+        <PlaceholderWrapper
```

Comment:
The placeholder is already inside an interactable `label`, so why would we need another button inside?